types/options.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface Options {
+    readonly fields?: string[],
+    readonly eager?: boolean,
+    readonly eagerL?: boolean,
+    readonly map?: boolean,
+    readonly rules?: boolean,
+    readonly meta?: boolean,
+    readonly build?: boolean,
+    readonly fill?: boolean,
+    readonly resolveA?: boolean,
+    readonly skip?: number,
+    readonly limit?: number,
+    readonly sort?: unknown[][],
```

Comment:
Previous `Options` used type `Array<Object>` which seems to me to be a bit different, no?
Because a value of `[{...fields}]` can be a `Array<Object>` but not a `unknown[][]`, if im not mistaken.

Reply:
It was wrong, in yonius it receives an array of arrays which is later iterated and a `sortObject` is constructed:
https://github.com/hivesolutions/yonius/blob/dac7219d73c45a56e2ad58459e071060dcd17227/js/data/model.js#L255-L258

Reply:
Ok but then it is always an array of arrays. Just the original type was wrong, ok!